### DISCUSSION TITLE
Blackduck Image for Nuget Codebase

### DISCUSSION THREAD

**Discussion by ajmalhot** [philips-software/docker-blackduck#173] [General]
Hi All,

Can I use the dotnetcore dockerfile to create an image for Nuget Codebase?

**Comment by JeroenKnoops** [comment]
You can, but I would recommend using https://github.com/synopsys-sig/detect-action

We're keeping this images for legacy projects. In newer projects we're moving towards using the GitHub action created by the vendor.

**Reply by ajmalhot** [reply]
Thanks for your reply! This is for legacy project. Do I need to install nuget on top of the dotnet base image? Can you please provide if you have any reference for nuget?
